DropDownButton: Add Overview (#7753)
DevExpress/devextreme-documentation: concepts/05 UI Components/DropDownButton/03 Overview/10 Key Features.md

- **Action Binding**    

> DropDownButton can trigger actions when users click drop-down list items ([onItemClick](/Documentation/ApiReference/UI_Components/dxDropDownButton/Configuration/#onItemClick)) or the DropDownButton itself ([onButtonClick](/Documentation/ApiReference/UI_Components/dxDropDownButton/Configuration/#onButtonClick)). You can also specify individual [click handlers](/Documentation/ApiReference/UI_Components/dxDropDownButton/Configuration/items/#onClick) for each item within the component data source.

Reviewer: ```suggestion
DropDownButton can trigger actions when users click drop-down list items ([onItemClick](/Documentation/ApiReference/UI_Components/dxDropDownButton/Configuration/#onItemClick)) or DropDownButton ([onButtonClick](/Documentation/ApiReference/UI_Components/dxDropDownButton/Configuration/#onButtonClick)). You can also specify individual [click handlers](/Documentation/ApiReference/UI_Components/dxDropDownButton/Configuration/items/#onClick) for each item within the component data source.
```

Author: I feel removing "the DropDownButton itself" changes this sentence for the worse. I think this is especially well illustrated if we shorten the text between "click" and "DropDownButton":

"DropDownButton can trigger actions when users click (something) or DropDownButton."

I propose a rewrite to get rid of "itself":
```suggestion
DropDownButton can trigger actions on button click ([onButtonClick](/Documentation/ApiReference/UI_Components/dxDropDownButton/Configuration/#onButtonClick)) or list item click ([onItemClick](/Documentation/ApiReference/UI_Components/dxDropDownButton/Configuration/#onItemClick)). You can also specify individual [click handlers](/Documentation/ApiReference/UI_Components/dxDropDownButton/Configuration/items/#onClick) for each item within the component data source.
```

Reviewer: I agree, that looks better

> - **Labeling**    

Reviewer 2: Feature list (names can be changes of course, I just want to show meaning of groups)
1. DropDownButton Appearance: 
- Icon
- Text Label 
- Dropdown arrow & showArrowIcon (meaning it can be hidden as well)
2. Items Appearance or Config or Settings
- icon
- text
- badge
3. Split Button Support
4. Predefined styling modes: 'text' | 'outlined' | 'contained' and  'danger' | 'default' | 'normal' | 'success'
5. Selection
6. A11y & KBN && RTL 
7. Customization

Reviewer 2: I don't think the Action Binding section is required here, because this functionality is obvious. But if you want to have it here we can add it before Customization.

But please change 
**items**.[onClick] 
-> 
items[].onClick(/Documentation/ApiReference/UI_Components/dxDropDownButton/Configuration/items/#onClick)

Author: Customers can set up actions to be triggered either on button or dropdown item click, which I think is a pretty important feature to mention. Moreover, customers can set up click handlers for each individual item or one common one for all items. Consequently, I think this point is important enough to place either in second or third place from the top.

Author: Following a discussion with Vlada, we decided to place "Action Binding" in first place. Please review and share your thoughts on this.

    Executed when users click any DropDownButton item.
    - **items**.[onClick](/Documentation/ApiReference/UI_Components/dxDropDownButton/Configuration/items/#onClick)    
    Executed when users click a specific DropDownButton item.

- **Split Button Support**    
Enable [splitButton](/Documentation/ApiReference/UI_Components/dxDropDownButton/Configuration/#splitButton) to separate the dropdown and action buttons of the DropDownButton component.

- **Selection**    
DevExtreme DropDownButton supports selection. If [useSelectMode](/Documentation/ApiReference/UI_Components/dxDropDownButton/Configuration/#useSelectMode) is enabled, the component displays the text and icon of the selected item. DropDownButton includes the following selection properties:
    - [selectedItem](/Documentation/ApiReference/UI_Components/dxDropDownButton/Configuration/#selectedItem)
    - [selectedItemKey](/Documentation/ApiReference/UI_Components/dxDropDownButton/Configuration/#selectedItemKey)
    - [onSelectionChanged](/Documentation/ApiReference/UI_Components/dxDropDownButton/Configuration/#onSelectionChanged)

- **Customization**    

> You can customize the appearance of DevExtreme DropDownButton elements as your needs dictate. Specify the following properties to implement your customizations:

Reviewer 2: Let's leave here only templates but please add a short description what each template is for.

    - [stylingMode](/Documentation/ApiReference/UI_Components/dxDropDownButton/Configuration/#stylingMode)
    - [type](/Documentation/ApiReference/UI_Components/dxDropDownButton/Configuration/#type)
    - [template](/Documentation/ApiReference/UI_Components/dxDropDownButton/Configuration/#template)
    - [itemTemplate](/Documentation/ApiReference/UI_Components/dxDropDownButton/Configuration/#itemTemplate)

>     - **items**.[template](/Documentation/ApiReference/UI_Components/dxDropDownButton/Configuration/items/#template)

Reviewer 2: **items**.[template](/Documentation/ApiReference/UI_Components/dxDropDownButton/Configuration/items/#template)
->
[items[].template](/Documentation/ApiReference/UI_Components/dxDropDownButton/Configuration/items/#template)
